index.html

```diff
@@ -92,6 +94,7 @@ <h3>
     <footer class="app-grid__footer">
       <nav class="nav">
         <a class="app-grid__link" href="home"><i class="fas fa-home"></i></a>
```

Comment:
This element is within your navigation, the correct usage of BEM here would be:
```suggestion
        <a class="nav__link" href="home"><i class="fas fa-home"></i></a>
```